snapshot/Chart.yaml

```diff
@@ -14,7 +14,7 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.1.0
+version: 0.2.0
```

Comment:
i think just a single increment to `version: 0.1.0` is okay. we haven't released the changes yet.